<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <action name="actionZoom_in">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/zoom_in.png</normaloff>:/buttons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/zoom_out.png</normaloff>:/buttons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionPan_x_y">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/hand_xy.png</normaloff>:/buttons/hand_xy.png</iconset>
   </property>
   <property name="text">
    <string>Pan x/y</string>
   </property>
   <property name="toolTip">
    <string>Pan x/y</string>
   </property>
  </action>
  <action name="actionPan_z">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/hand_xz.png</normaloff>:/buttons/hand_xz.png</iconset>
   </property>
   <property name="text">
    <string>Pan z</string>
   </property>
   <property name="toolTip">
    <string>Pan z</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/rotate.png</normaloff>:/buttons/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/buttons/cursor.png</normaloff>:/buttons/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
